File: src/cli/discord_command.zig
pub const DiscordCommand = struct {
    const discord_url = "https://bun.com/discord";
    pub fn exec(_: std.mem.Allocator) !void {
        open.openURL(discord_url);
    }
};

const bun = @import("bun");
const open = @import("../open.zig");
const std = @import("std");

File: src/runtime/cli/discord_command.rs
use crate::cli::open;

pub(crate) struct DiscordCommand;

impl DiscordCommand {
    const DISCORD_URL: &'static [u8] = b"https://bun.com/discord";

    // Infallible body; `Result` only matches the CLI command dispatch signature.
    pub(crate) fn exec() -> Result<(), crate::Error> {
        open::open_url(Self::DISCORD_URL);
        Ok(())
    }
}
